packages/core/src/indexing/index.test.ts

```diff
@@ -59,20 +51,21 @@ test("createIndexing()", async (context) => {
     schemaBuild: { schema },
   });
 
+  const cachedViemClient = createCachedViemClient({
+    common,
+    indexingBuild: { networks },
+    requestQueues: [createRequestQueue({ network: networks[0]!, common })],
```

Comment:
Is it an invariant that the indexes of the networks and requestQueues arrays are the same?

Reply:
yes